orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/KatoRestService.groovy

```diff
@@ -45,6 +46,11 @@ interface KatoRestService {
                            @Path("cloudProvider") String cloudProvider,
                            @Body Collection<? extends Map<String, Map>> operations)
 
+  @POST("/{cloudProvider}/ops")
+  Response submitOperations(@Query("clientRequestId") String clientRequestId,
+                            @Path("cloudProvider") String cloudProvider,
+                            @Body Collection<Map<String, OperationContext>> operations);
```

Comment:
Not really different from the other `submitOperations` methods, except this one returns the `Response`, as the response code is significant for this stage to work correctly.

Reply:
Thoughts on whether this should support the list of operations model here rather than just hitting a single operation `/{cloudProvider}/ops/{operationName}`?

I think the most we've used the ability to submit a list of ops for was to do things like injecting an allow launch before an AWS deploy and things like that which will move to clouddriver with these changes

It makes it harder to reason about what the Response code means when there are multiple operations in the payload.

Not sure if you have some specific cases in mind where we might need to submit multiple ops to clouddriver?

Reply:
Oh, no. I was just trying to keep consistent with existing use cases. I'd love to see multiple ops disappear. Nice! I'll switch over to that.